ldk/javascript/src/whisper/form.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { Components, Form, Whisper, WhisperComponentType } from '.';
+
+export class LdkForm {
+    children: Array<Components>;
+
+    componentState: Map<string, any>;
+
+    constructor(children: Array<Components>) {
+        this.componentState = new Map<string, any>();
+        this.children = children;
+
+        this.setInitialComponentState(children);
+        this.registerListeners(children);
+    }
+
+    getComponentState(): Map<string, any> {
+        return this.componentState;
+    }
+
+    private isOnChangeInput(child: any): boolean {
+        return (child.onChange !== undefined && child.name !== undefined);
+    }
+
+    private isOnSelectInput(child: any): boolean {
+        return (child.onSelect !== undefined && child.name !== undefined);
```

Comment:
Will it be the same to have: (!child.onSelect && !child.name); ?

Reply:
My function is `isOnChangeInput` so I believe I can change to `(child.onSelect && child.name)` where both need to evaluate as truthy.